### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by hlcampos** [Prodigy-Hacking/ProdigyMathGameHacking#1338] [Questions]
i am guilty of hacking please make a button to take back all hacks please

**Comment by afkvido** [comment]
Hi!
You can use the **Reset Account** hack to completely reset your account.

If that's not what you want, you can also use the **Remove Item** and **Remove Pet** hacks to delete items/pets.

**Reply by hlcampos** [reply]
thanks

**Comment by ghost** [comment]
# Also, it said to use an ALT ACCOUNT
If you used your primary account, that is not very wise.
If you did use an alt account, just don't use it if you're guilty.

**Reply by afkvido** [reply]
LMAO 

https://github.com/Prodigy-Hacking/ProdigyMathGameHacking#disclaimer

**Reply by hlcampos** [reply]
i dident read it

**Reply by afkvido** [reply]
rip

**Reply by hlcampos** [reply]
is it possible to contact the creator of the extension

**Reply by afkvido** [reply]
bro that's easy

**Reply by hlcampos** [reply]
then tell me

**Reply by hlcampos** [reply]
waiting

**Reply by hlcampos** [reply]
is it long

**Reply by hlcampos** [reply]
huuuuuuuuuuuuuaaaaaaaaaaaaaaaaah

**Comment by hlcampos** [comment]
contact the creator

**Reply by afkvido** [reply]
@ArcerionDev

**Reply by ghost** [reply]
It thought [PatheticMustan](https://github.com/patheticmustan/) is the creator.

**Reply by afkvido** [reply]
but arceriondev is the current owner?

**Reply by ghost** [reply]
> "I'm also the creator of Prodigy-Hacking/ProdigyMathGameHacking. For proof, PatheticMustan/ProdigyMathGameHacking redirects to the official Prodigy-Hacking repo."

-PatheticMustan

**Reply by afkvido** [reply]
True, Mustan is the creator.

but arse is the current owner

**Reply by afkvido** [reply]
also , you can do

`\- PatheticMustan`

**Reply by afkvido** [reply]
because backslash

**Reply by ghost** [reply]
\- patheticmustan

**Reply by PatheticMustan** [reply]
> True, Mustan is the creator.
> 
> but arse is the current owner

we're co-owners.


> i am guilty of hacking please make a button to take back all hacks please

unfortunately there's no clean "remove all hacks" button, the only thing you can really do is removing pets/items, or resetting your account.
